documents/rulebook.tex

```diff
@@ -73,7 +73,6 @@ \chapter{Tests in Stage I}
 \testtocentry{test:gpsr}
 \testtocentry{test:receptionist}
 \testtocentry{test:serve-breakfast}
-\testtocentry{test:storing-groceries}
```

Comment:
Just add the combined task and don't remove storing groceries for now.

Reply:
Fixed